### PR TITLE
CleverStopwords added unit test

### PR DESCRIPTION
**Pull request by adrianne-l**

### CODE REVIEW

File: src/clevercloud/CleverStopwords.py [hunk 1]
```diff
@@ -23,13 +23,27 @@ def CleverStopwords(words):
...
+        if not isinstance(item, str):
+            raise TypeError("Values of input variable should be a string.")
+
     # NLTK English stopwords
     stopwords = set(stopwords.words("english"))
 
+    try:
+        nltk.download("stopwords")
+    except:
+        raise ImportError("Stopwords are not downloaded from NLTK.")
```
Comment (ajuliettet) [inline src/clevercloud/CleverStopwords.py:45]: Will this part of the code check whether nltk.download("stopwords") exist?

File: tests/test_CleverStopwords.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# author: Adrianne Leung
+# date: Jan 2022
+
+from clevercloud.CleverStopwords import CleverStopwords
+import pytest
```
Comment (ajuliettet) [inline tests/test_CleverStopwords.py:5]: Should we import packages within or outside a test function?

File: src/clevercloud/CleverStopwords.py [hunk 3]
```diff
@@ -23,13 +23,27 @@ def CleverStopwords(words):
...
     nltk.download('stopwords')
+    
+    # Test input variable type
     if not isinstance(words, set):
         raise TypeError("Input variable should be a set.")
 
+    # Test data type of values of input variable
+    for item in words:
+        if not isinstance(item, str):
+            raise TypeError("Values of input variable should be a string.")
```
Comment (ajuliettet) [inline src/clevercloud/CleverStopwords.py]:
```suggestion
            raise TypeError("Each element of the input should be a string.")
```

**Review by ajuliettet — CHANGES_REQUESTED** [review]

File: src/clevercloud/CleverStopwords.py (hunk 1 above)
Comment (adrianne-l) [reply src/clevercloud/CleverStopwords.py:45]: It is to check whether the stopwords library is downloaded successfully from NLTK to use as the default stopwords list.

File: tests/test_CleverStopwords.py (hunk 2 above)
Comment (adrianne-l) [reply tests/test_CleverStopwords.py:5]: To align with all files, I'll move the import nltk outside of the function.

File: src/clevercloud/CleverStopwords.py (hunk 1 above)
Comment (ajuliettet) [reply src/clevercloud/CleverStopwords.py:45]: Great. Thanks!

**Review by Arushi282 — APPROVED** [review]

**Review by ajuliettet — APPROVED** [review]

**Pull request merged**